app/view_objects/start_page.rb

```diff
@@ -2,11 +2,7 @@ class StartPage
   attr_accessor :news, :events, :notices
 
   def initialize(member: false)
-    @news = News.includes(:translations).
-      includes(:user).
-      includes(:categories).
-      order(created_at: :desc).
-      limit(5) || []
+    @news = News.include_for_feed.by_date.limit(5) || []
```

Comment:
Heh - perhaps the hound means:

_It is missing a space for the operator_ **OR** _it is just a dot_